components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/util/APIErrorResponseHandler.java

```diff
@@ -0,0 +1,160 @@
+package org.wso2.carbon.identity.auth.valve.util;
```

Comment:
License header missing

Reply:
This has been fixed with 747c2c448717dcac5a09b0abec9838c5d61ac644